pysgrid/sgrid.py

```diff
@@ -514,14 +525,15 @@ def save_as_netcdf(self, filepath):
             # populate variables with data
             grid_time[:] = self.grid_times[:]
             grid_center_lon[:] = self.centers[..., 0]
-            grid_center_lat[:] = self.centers[..., 1] 
-            grid_node_lon[:] = self.nodes[..., 0]
-            grid_node_lat[:] = self.nodes[..., 1]
+            grid_center_lat[:] = self.centers[..., 1]
+            if self.nodes is not None: 
+                grid_node_lon[:] = self.nodes[..., 0]
+                grid_node_lat[:] = self.nodes[..., 1]
```

Comment:
Nothing would bring a smile to my face more than terminating support for `SGrid3D`. :smile: I'm still waiting to for @hrajagers to weigh-in before making and dramatic changes to the 3D grids. I don't have much desire to 'cowboy' SGrid3D's removal from master before a formal decision is made. 

I like simpler code too; it simplifies development, and reduces maintenance and technical debt burdens.